CONTRIBUTING.md

```diff
@@ -137,23 +137,22 @@ Committers!
 With great power comes great responsibility yada yada yada:
 
  * Code is peer reviewed, you should (almost) never push your own code.
- * Please don't accidentally force push to master.
- * Cherry Pick / Rebase commits, **don't use the big green button**, see below for instructions on how to
+ * Pushing to `master` is blocked by GitHub.
+ * Cherry Pick / Rebase commits, **don't use merge commits**, see below for instructions on how to
  merge a pull request.
  * Ensure reviewed code follows the above contribution guidelines, if it doesn't feel free to amend and make note.
  * Please try to watch when Pull Requests are made and review and / or commit them in a timely manner.
  * After you merge in a patch use tin to update the version accordingly. Run `tin -v x.x.x-prerelease` with x.x.x being the previous version upgraded appropriately via semver. When we are ready to publish to npm we can remove the `-prerelease`.
```

Comment:
is this still followed? it seems now the updates to versions are only made in a "about to make a release phase" not on every merge...

Reply:
If it _is_ used, it's probably `npx tin` btw.

Reply:
decided to merge this in allready while wating for someone to confirm current usage of tin